src/Components/StatusDelegator/index.js

```diff
@@ -25,18 +26,25 @@ const KeyValueStyled = styled(KeyValue)`
 `
 
 const StatusDelegator = props => {
-  const { summary } = props
+  const { summary, web3 } = props
   const { totalStakeInLPT, fees, status } = summary
+  let feesInETH = 0
+  // Formats fees from wei to ETH
+  if (fees) {
+    feesInETH = web3.utils.fromWei(fees.toString(), 'ether')
+  }
+
   const tableData = [
     {
       data: totalStakeInLPT,
       text: 'LPT Staked',
       tooltip: toolTipsTexts.TOTAL_STAKE_TOOLTIP,
     },
     {
-      data: fees,
+      data: toFixedDecimals(feesInETH, 2),
       text: 'ETH Earning fees',
       tooltip: toolTipsTexts.EARNING_FEES_TOOLTIP,
+      tooltipDataText: feesInETH,
```

Comment:
I think it would be better to group the tooltip data into an object, to avoid confusion with the other properties and inconsistency in the names, something like this `
tooltip: {
  text: '...'
  value: '...'
}
`